src/services/track/admin/subscription-submitter.ts

```diff
@@ -152,37 +152,40 @@ export class SubscriptionSubmitter implements IObserver {
 				this.stripe.customers.retrieve(data.paymentProviderId),
 			]);
 			if (!customer) {
-				const whereClause: FindOptionsWhere<CustomerEntity>[] = [{ paymentProviderId: data.paymentProviderId }];
-				// we add an additional "OR" check in case that a customer was created locally with email and no paymentProviderId
-				if (!stripeCustomer.deleted && stripeCustomer.email) {
-					whereClause.push({ email: stripeCustomer.email });
-				}
-
 				const customers = await CustomerService.instance.customerRepository.find({
```

Comment:
@jay-dee7 Can we just use findByPaymentProviderId here. Do we need to check for duplicate paymentProvidersId at this step? I think we can do that in a customer created event right?